thirdparty/LICENSE

```diff
@@ -0,0 +1,283 @@
+The SINGA project contains subcomponents with separate copyright
```

Comment:
I don't think it is required to separate the license in multiple files. See MongoDB third party license for example:

https://github.com/mongodb/mongo/blob/master/distsrc/THIRD-PARTY-NOTICES